newIDE/app/src/SceneEditor/index.js

```diff
@@ -132,6 +133,7 @@ type Props = {|
   onChooseResource: ChooseResourceFunction,
   resourceExternalEditors: Array<ResourceExternalEditor>,
   isActive: boolean,
+  layoutVariablesDialogOpen: boolean,
```

Comment:
Do you still need this?

Reply:
Yeah. It set's the current layoutVariablesDialogOpen in the SceneEditor state to use the layoutVariablesDialogOpen in the props as default value if layoutVariablesDialogOpen is passed in as props